components/server/src/authorization/relationship-updater.spec.db.ts

```diff
@@ -71,6 +69,10 @@ describe("RelationshipUpdater", async () => {
     });
 
     it("should update a simple user once it was feature-flag disabled", async () => {
```

Comment:
I don't have much context here, but it looks like it's alright to just remove this instead of skipping when the feature flag will not exist anymore? It's not that big of a deal to keep, but curious what you had in mind 🙏

Reply:
I thought that too, but opted to keep it for illustration purposes, e.g. that we have (and maintain) that feature.